### PR TITLE
Use 1ES pool for OneBranch test pipeline

### PR DESCRIPTION
**Pull request by csujedihy**
## Description

No more office hosted test machines.

## Testing

OB CI

### CODE REVIEW

File: .azure/OneBranch.Tests.Official.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ trigger: none # https://aka.ms/obpipelines/triggers
 resources:
   pipelines:
   - pipeline: onebranch   # Name of the pipeline resource
-    source: msquic-Official # Name of the pipeline referenced by the pipeline resource
+    source: msquic-1es-pool-azp-Official # Name of the pipeline referenced by the pipeline resource
```
Comment (csujedihy) [inline .azure/OneBranch.Tests.Official.yml:6]: oops

**Review by nibanks — COMMENTED** [review]
You already tested this?

**Comment by csujedihy** [comment]
> You already tested this?

Oops, brainfart. This is for internal repo rather than AZP perf. This is not ready yet.

**Comment by csujedihy** [comment]
> > You already tested this?
> 
> Oops, brainfart. This is for internal repo rather than AZP perf. This is not ready yet.

This has been tested working in Onebranch.

**Review by nibanks — APPROVED** [review]

**Pull request merged**